packages/core/src/Stores/ui-store.js

```diff
@@ -26,7 +26,8 @@ export default class UIStore extends BaseStore {
     settings_extension = undefined;
     notification_messages_ui = undefined;
 
-    is_dark_mode_on = window?.matchMedia?.('(prefers-color-scheme: dark)').matches && isMobile();
```

Comment:
if user requested a theme, why don't we provide it to them?
ref: https://developer.mozilla.org/en-US/docs/Web/CSS/@media/prefers-color-scheme

Reply:
@matin-deriv I left the similar [comment](https://github.com/binary-com/deriv-app/pull/12012#pullrequestreview-1764295723), but you have to ask someone who created this card, but not the developer :)

Reply:
@sergei-deriv The pull requests are the responsibility of the developers, and we, as developers, should not undertake tasks without a clear understanding. Ultimately, we are accountable for the implementations, and creating a pull request should be the final step once we are fully aware of the expectations.